### PR TITLE
Add /notifyuser

### PR DESCRIPTION
**Pull request by PartMan7**
See [here](https://www.smogon.com/forums/threads/only-modnote-notifyrank-if-it-highlights-everyone.3687633/).

### CODE REVIEW

File: server/chat-commands/moderation.ts [hunk 1]
```diff
@@ -1636,6 +1636,33 @@ export const commands: Chat.ChatCommands = {
...
+			room.sendUser(targetUser, `|tempnotifyoff|${id}`);
+		} else {
+			let [title, notification] = this.splitOne(titleNotification);
+			if (!title) title = `${room.title} notification!`;
+			if (!user.can('addhtml')) {
+				title += ` (notification from ${user.name})`;
+			}
+			if (notification.length > 300) return this.errorReply(`Notifications should not exceed 300 characters.`);
+			const message = `|tempnotify|${id}|${title}|${notification}`;
+			room.send(message);
```
Comment (AnnikaCodes) [inline server/chat-commands/moderation.ts:1658]: Shouldn't this only be sent to the user, with `Room#sendUser`?

Comment (PartMan7) [reply server/chat-commands/moderation.ts:1658]: Right, whoops

I'd only tested with two users in the room

**Comment by AnnikaCodes** [comment]
This seems fine, thanks!

**Review by AnnikaCodes — APPROVED** [review]
I'll wait a few days for others to comment and then merge.

**Comment by Zarel** [comment]
Looks good to me, too.

**Review by Zarel — APPROVED** [review]

**Pull request merged**